vector/src/main/res/values/strings.xml

```diff
@@ -1462,11 +1462,21 @@
     <string name="room_settings_room_access_entry_knock">Anyone can knock on the room, members can then accept or reject</string>
     <string name="room_settings_room_access_entry_unknown">Unknown access setting (%s)</string>
     <string name="room_settings_room_access_private_title">Private</string>
+    <string name="room_settings_room_access_private_invite_only_title">Private (Invite Only)</string>
     <string name="room_settings_room_access_private_description">Only people invited can find and join</string>
     <string name="room_settings_room_access_public_title">Public</string>
     <string name="room_settings_room_access_public_description">Anyone can find the room and join</string>
-    <string name="room_settings_room_access_restricted_title">Spaces</string>
+    <string name="room_settings_room_access_restricted_title">Space members only</string>
     <string name="room_settings_room_access_restricted_description">Anyone in a space with this room can find and join it. Only admins of this room can add it to a space.</string>
+    <string name="room_create_member_of_space_name_can_join">Members of Space %s can find, preview and join.</string>
+    <string name="allow_space_member_to_find_and_access">Allow space members to find and access.</string>
+    <string name="spaces_which_can_access">Spaces which can access</string>
+    <string name="decide_which_spaces_can_access">Decide which spaces can access this room.  If a space is selected its members will be able to find  and join Room name.</string>
```

Comment:
Too many spaces in `room. If` and `find and`

Reply:
done